drivers/docker/utils.go

```diff
@@ -237,7 +237,7 @@ func parseVolumeSpecWindows(volBind string) (hostPath string, containerPath stri
 		return "", "", "", fmt.Errorf("not <src>:<destination> format")
 	}
 
-	hostPath = parts[0]
+	hostPath = filepath.FromSlash(parts[0])
```

Comment:
Can you add a comment here explaining why FromSlash is useful here?

Reply:
can you help me phrase it, this is what i got
// for nomad to be OS agnostic we allow to supply the host mount path with forward or backward slash
// we then convert the slashes to OS specific slash in the agent side

Reply:
@schmichael update the phrasing again